html/mime-analysis.html

```diff
@@ -0,0 +1,147 @@
+                            <li class="dropdown-header">About Us</li>
+                            <li><a class="page-scroll" href="../index.html#about">What Is Polar Data Insights?</a></li>
+                            <li><a class="page-scroll" href="../index.html#features">Visualization Features</a></li>
+                            <li role="separator" class="divider"></li>
+                            <li><a class="page-scroll" href="../index.html#started">Getting Started</a>
+                        </ul>
+                    </li>
+                    
+                    <!--TODO: Change this to polar.usc.edu/solr once it is up and running -->
+                    <li>
+                    <!--    <a class="page-scroll" href=" html/placeholder.html " style="color:#ffffff">Search</a>-->
+
+
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" style="color:#ffffff">Search <span class="caret"></span></a>
+                        <ul class="dropdown-menu">
+                            <li class="dropdown-header">Search</li>
+                            <li><a class="page-scroll" href="http://polar.usc.edu/Polar/banana/#/dashboard"> Banana </a></li>
+                            <li><a class="page-scroll" href="http://polar.usc.edu/Polar/facetview/">FacetView</a></li>
+                    </li>
+                        </ul>
+                    </li>
+                    <li><a style="color:#ffffff" class="page-scroll" href="d3-examples.html">Visualizations</a>
+                    <li>
+                        <a class="page-scroll" href="http://irds.usc.edu/" style="color:#ffffff">USC Data Science Group</a>
+                    </li>
+                    <li><a class="page-scroll" style="color:#ffffff" href="./mime-analysis.html">MIME Analysis</a></li>
+
+
+                </ul>
+                <a href="http://usc.edu" class="pull-right">
+                  <img src="../images/usc-primary-logotype.svg" width="176px" height="38px">
+                </a>
+            </div>
+            <!-- /.navbar-collapse -->
+        </div>
+        <!-- /.container -->
+    </nav>
+
+    
+
+    
+
+    <section id="D3" class="D3-section">
+        <div class="container">
+            <div class="row">
+                <div class="row" style="margin-top:10%">
+
+                    <!-- Note: each image is 242x200  -->
+                    <div class="col-sm-6 col-md-4">
+                        <div class="thumbnail">
+                            <img src="../images/team28.png">
+                            <div class="caption">
+                                <h3>MIME Analysis Team 28</h3>
+                                <p>These visualisations contribute to explaning the MIME Diversity in Polar Dataset.</p>
+                                <p><a href="http://rahulravindran0108.github.io/Polar-Dataset-Explorer/" class="btn btn-primary" role="button">View</a>
```

Comment:
can you replace this external link with the actual visualizations and add them here to the pull request?